CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Release 0.25.6
+
+* Enhancement: Updated Control Channel to use new heartbeat logging mirroring Links in Release `0.25.0`
+* Enhancement: Added Circuit Creation Timespan which denotes how long the fabric took to construct a requested circuit.
```

Comment:
Can you add a quick example of what the JSON output looks like so ops knows what to expect

Reply:
Added an example json output